source/Files.h

```diff
@@ -38,6 +38,10 @@ class Files {
 	static const std::string &Sounds();
 	static const std::string &Saves();
 	static const std::string &Tests();
+
+	static std::string GetBasePath();
```

Comment:
Well I can't move them to Files.cpp or else I would need to duplicate the whole class definition. All those mocks are a bit hacky and I don't like them.

They are a bit of a maintenance nightmare. I think instead of replacing our code, I provide dummy implementations of all our libraries. Less maintenance and cleaner code probably. I'll do that tomorrow.